xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -733,6 +733,23 @@ int CEdl::GetTotalCutTime() const
   return m_totalCutTime; // ms
 }
 
+const std::vector<EDL::Edit> CEdl::GetEditList() const
+{
+  std::vector<EDL::Edit> editList;
+  for (EDL::Edit edit : m_vecEdits)
```

Comment:
You are taking a copy of the vector member and later on you modify the copy. Is this intended?

Reply:
Yeah it is intended but I have a few doubts. I don't want to modify the original edit - hence the copy to modify later. If I was to iterate as a ref I'd risk modifying the original edit isn't it?

Reply:
There are other places I changed that could well return const ref, will change in next revision. However here, it is intended to be a copy

Reply:
> However here, it is intended to be a copy
 
 That was my impression too. Just wanted to double-check with you. Maybe a comment to make clear this is by intention would be good?

Reply:
done